src/solvers/dgsem_tree/indicators.jl

```diff
@@ -212,6 +212,85 @@ end
 
 
 
+"""
+    IndicatorIDP(equations::AbstractEquations, basis;
+                 IDPPositivity=false,
+                 variables_cons=(first,),
+                 positCorrFactor=0.1)
+
+Blending indicator used for subcell shock-capturing [`VolumeIntegralShockCapturingSubcell`](@ref) including:
+- positivity limiting for conservative variables.
+
+## References
+
+- Rueda-Ramírez, Pazner, Gassner (2022)
+  "Subcell Limiting Strategies for Discontinuous Galerkin Spectral Element Methods"
+- Pazner (2020)
+  "Sparse invariant domain preserving discontinuous Galerkin methods with subcell convex limiting"
+  [arXiv:2004.08503](https://doi.org/10.1016/j.cma.2021.113876)
+
+!!! warning "Experimental implementation"
+    This is an experimental feature and may change in future releases.
+"""
+struct IndicatorIDP{RealT<:Real, LimitingVariablesCons, Cache} <: AbstractIndicator
+  IDPPositivity::Bool
+  variables_cons::LimitingVariablesCons # Positivity of conservative variables
+  cache::Cache
+  positCorrFactor::RealT                # Correction factor for IDPPositivity
```

Comment:
```suggestion
  positivity_correction_factor::RealT                # Correction factor for IDPPositivity
```
It really helps understandability to avoid abbreviations. For example, later you refer to this as the "correlation factor" (thus either this comment or the later code place is wrong). Also, please always use `snake_case` for variables